src/index.tsx

```diff
@@ -1,21 +1,29 @@
-import * as React from "react";
-import { useState, useEffect, useMemo, useRef } from "react";
+import React, { useCallback, useReducer, useRef } from "react";
+import { useState, useEffect } from "react";
 import { createRender, useModelState, useModel } from "@anywidget/react";
 import type { Initialize, Render } from "@anywidget/types";
 import Map from "react-map-gl/maplibre";
 import DeckGL, { DeckGLRef } from "@deck.gl/react/typed";
-import { PolygonLayer } from "@deck.gl/layers/typed";
-import type { PickingInfo } from "@deck.gl/core/typed";
-import { MapViewState, type Layer } from "@deck.gl/core/typed";
+import { MapViewState, PickingInfo, type Layer } from "@deck.gl/core/typed";
 import { BaseLayerModel, initializeLayer } from "./model/index.js";
 import type { WidgetModel } from "@jupyter-widgets/base";
 import { initParquetWasm } from "./parquet.js";
 import { getTooltip } from "./tooltip/index.js";
-import { makePolygon, isDefined, loadChildModels } from "./util.js";
+import { isDefined, loadChildModels, rateLimit } from "./util.js";
 import { v4 as uuidv4 } from "uuid";
 import { Message } from "./types.js";
 import { flyTo } from "./actions/fly-to.js";
 import { useViewStateDebounced } from "./state";
+import * as selectors from "./reducer/selectors";
+
+import "maplibre-gl/dist/maplibre-gl.css";
+
+import {
+  ActionTypes,
+  baseInitialState,
+  MapMode,
+  reducer,
+} from "./reducer/index.js";
```

Comment:
Move this next to previous reducer import?

Reply:
Done.